packages/core/stories/about/Usage.stories.mdx

```diff
@@ -1,20 +1,40 @@
 import { Meta } from '@storybook/addon-docs/blocks';
 import { linkTo } from '@storybook/addon-links';
-import {
-  Button
-} from '@massds/mayflower-react';
+import { Button } from '@massds/mayflower-react';
 
 import generateTitle from '../util/generateTitle';
 const { STORYBOOK_CDN_PATH, STORYBOOK_CDN, STORYBOOK_PKG } = process.env;
 
 
 <Meta title={generateTitle('Usage')} />
 
-# Install Mayflower
+<h1>Get Started with Mayflower <small>version {process.env.VERSION}</small></h1>
+
+
 
 We encourage all Commonwealth web properties to implement the Mayflower Design System for a consistent look and feel. It’s important that constituents know they are on an official state government website, and the visual cues that come from using a cohesive design help convey that legitimacy.
 
-## Mayflower assets
+
+## Quick Start
+To kickstart an HTML page with Mayflower Header, Footer, basic layout and styles:
+<Button theme="c-primary-alt" onClick={linkTo(generateTitle('Template'))}>Start From a Template</Button>
+
+To start building with Mayflower in HTML, start by importing the global styles and then incrementally add the styles and JS for each Mayflower component:
+
+- To import fonts and get started with basic Mayflower styles, link to the compiled `css/global.css` (or the minified version, `css/global.min.css`) from the @massds/mayflower-assets package or the assets CDN.
```

Comment:
Shouldn't we provide the full CDN links here and on 120?

Reply:
sure